CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This is a changelog for Piwik platform developers. All changes for our HTTP API'
 
 ### New features 
 * The JavaScript Tracker does now track outlinks and downloads if a user opens the context menu if the `enabled` parameter of the `enableLinkTracking()` method is set to `true`. To use this new feature use `tracker.enableLinkTracking(true)` or `_paq.push(['enableLinkTracking', true]);`. This is not industry standard and is vulnerable to false positives since not every user will select "Open in a new tab" when the context menu is shown. Most users will do though and it will lead to more accurate results in most cases.
+* The JavaScript Tracker now contains the 'heart beat' feature which can be used to obtain more accurate visit lengths by periodically sending 'ping' requests to Piwik. To use this feature use `tracker.setHeartBeatTimer();` or `_paq.push(['setHeartBeatTimer']);`. By default, a ping request will be sent every 15 seconds. You can specify a custom time by passing an argument, eg, `tracker.setHeartBeatTimer(10);` or `_paq.push(['setHeartBeatTimer', 10]);`.
 
```

Comment:
> This shouldn't be necessary. If you have a tab open for an hour, ping requests will be sent but only if the tab is in focus. If you're on another window, nothing will be sent.

Yeah I thought so. It's certainly not MVP and if we notice some users become problems then we can still add it.

>> should be name it maybe enableHeartBeatTimer(heartBeatDelay)
> Sounds ok to me

See further below. Maybe even `startHeartBeatTimer(delay)` and `stopHeartBeatTimer()` would be better. From a `enableHeartBeatTimer()` method I would not really expect to have a parameter but can. Not sure what makes more sense :) Think I'd actually prefer `start/stop` or so